frontend/src/lib/k8s/api/v2/hooks.ts

```diff
@@ -356,7 +356,9 @@ function _useKubeObjectLists<K extends KubeObject>({
     if (items === null) {
```

Comment:
this block can be simplified a bit

```ts
const items = clusterResults.flatMap(result => result.items ?? [])
```

Reply:
No, that doesn't quite work. Because nullability of that data, and it also needs to iterate over clusters.